source/compatibility.txt

```diff
@@ -33,13 +33,23 @@ The first column lists the driver version.
 
 Language Compatibility
 ----------------------
-The following compatibility table specifies the recommended versions of
+
+The following compatibility table specifies the supported versions of
 the {+driver-long+} for use with a specific version of Node.js.
 
 The first column lists the driver version.
 
 .. include:: /includes/language-compatibility-table-node.rst
 
+.. note:: The {+driver-short+} Supports Node.js Versions in the "Active LTS" Phase or Later
+
+   After release, Node.js versions have a "Current" release status for six months 
```

Comment:
Q: does "Current" need to be capitalized?

Reply:
Capitalized it, as it is in the Node.js docs.